tests/test_api.py

```diff
@@ -1120,6 +1120,137 @@ def test_gracefully_handle_multiple_conditions_on_same_column(self):
         assert response.status_code == 200
 
 
+class TestTransactionsApi(BaseApiTest):
```

Comment:
please let's put this into a separate file.

Reply:
Ok :+1: